cluster/up.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+source ./cluster/gocli.sh
+
+$gocli run --random-ports --nodes 2 --background kubevirtci/k8s-1.10.3
```

Comment:
Need to just adjust this to take different providers and node counts based on the same KUBEVIRT_PROVIDER and KUBEVIRT_NODES variables, its hard coded here.